src/pages/catalog-page/CatalogPage.tsx

```diff
@@ -35,33 +32,26 @@ const paginationStyles = {
 }
 
 const CatalogPage = () => {
-  const { params, page } = useParams()
-  const [offset, setOffset] = useState<number>(parseInt(page!))
-  const [operationType, setOperationType] =
-    useState<FetchItemOperationType>('fetch')
+  const { page }: any = useParams()
+  const [pageN, setPageN] = useState(parseInt(page) || 1)
 
-  const allProducts = useSelector((state: RootState) => state.items.items)
+  const totalCount = useSelector((state: RootState) => state.items.totalCount)
+  const offset = useSelector((state: RootState) => state.items.offset)
   const dispatch = useDispatch<AppDispatch>()
-  const navigate = useNavigate()
-
-  useEffect(() => {
-    dispatch(fetchItems({ params: offset, operationType: operationType }))
-  }, [dispatch, offset, operationType])
+  const totalNumberOfPages = Math.ceil(totalCount / 9)
```

Comment:
what is `9` mean?

Reply:
9 is the number of products on one page, I put the value into a constant